### ISSUE TITLE
Clicking on labels opens unexpected plot

### ISSUE THREAD

**Issue by timmeh87** [custom-cards/dual-gauge-card#51]
I would expect to click on the right value to see the inner value plot, and the left value to see the outer plot.

However what really happens is that both the right and left values open the plot for the "inner" bar, and then clicking on the text label underneath opens the the plot for the "outer" bar.

**Comment by jaymunro** [comment]
I agree. It was not until I read the above that I now understand where to tap to get the pop up plots.
Until now I thought it was random and I just kept on tapping in frustration until I got the plot I was looking for.

It would be much more logical to get the plot of the value you tap on.

**Comment by timmeh87** [comment]
I switched to mini-graph-card, I found that gauges are cool but graphs are almost always better

**Comment by jaymunro** [comment]
Thanks for pointing that one out. It is indeed nice and I personally prefer the graphs too as the information is more dense, however as a result it is harder to see at a glance from a distance and the gauge is a simple construct for guests to understand.

**Comment by djoris** [comment]
similarly: `tap_action` seems unsupported.
